src/dali/plugin/input/rest/kucoin_futures.py

```diff
@@ -0,0 +1,288 @@
+# Copyright 2022 topcoderasdf
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import time
+from datetime import datetime
+from decimal import Decimal
+from typing import Any, Dict, List, Optional
+
+from ccxt import Exchange, RateLimitExceeded, kucoinfutures
+from dateutil import tz
+
+from dali.abstract_ccxt_input_plugin import AbstractCcxtInputPlugin
+from dali.ccxt_pagination import AbstractPaginationDetailSet
+from dali.configuration import Keyword
+from dali.in_transaction import InTransaction
+from dali.intra_transaction import IntraTransaction
+from dali.out_transaction import OutTransaction
+
+_AMOUNT: str = "amount"
+_CURRENCY: str = "currency"
+_DATA: str = "data"
+_DATA_LIST: str = "dataList"
+_DEPOSIT: str = "Deposit"
+_END_AT: str = "endAt"
+_HAS_MORE: str = "hasMore"
+_OFFSET: str = "offset"
+_REALISED_PNL: str = "RealisedPNL"
+_START_AT: str = "startAt"
+_TIME: str = "time"
+_TRANSFER_IN: str = "TransferIn"
+_TRANSFER_OUT: str = "TransferOut"
+_TS_INCREMENT: int = 86400000
```

Comment:
Should rename this to what it actually is:
```
_ONE_DAY_IN_MS: int = 86400000
```

Reply:
fixed